Makefile

```diff
@@ -73,9 +75,10 @@ betanet-test.wasm: target/wasm32-unknown-unknown/release/aurora_engine.wasm
 	cp $< $@
 
 target/wasm32-unknown-unknown/release/aurora_engine.wasm: Cargo.toml Cargo.lock $(shell find src -name "*.rs") etc/eth-contracts/res/EvmErc20.bin
-	RUSTFLAGS='-C link-arg=-s' $(CARGO) build \
+	RUSTFLAGS='-C link-arg=-s -C codegen-units=1 -C panic=abort' $(CARGO) build \
```

Comment:
For some reason the `codgen-units` and `panic` flags are not being picked up from the `Cargo.toml` file anymore. I added them back in explicitly here to fix the performance regression, but I'll continue to investigate why they are skipped in the cargo file to see if I can make it so these extra flags do not need to be given here.

Reply:
Fixed in https://github.com/aurora-is-near/aurora-engine/pull/243/commits/d674b85eb443caebf51f97b4ac9112f14c3f0a2e

The issue was that cargo profiles must always be specified in the root `Cargo.toml` file.